packages/vsce/package.json

```diff
@@ -19,10 +19,38 @@
   "contributes": {
     "commands": [
       {
-        "command": "zowe-native-proto-vsce.helloWorld",
-        "title": "Hello World"
+        "command": "zowe-native-proto-vsce.connect",
+        "title": "Zowe-SSH: Connect to Host..."
+      },
+      {
+        "command": "zowe-native-proto-vsce.restart",
+        "title": "Zowe-SSH: Restart Zowe Server on Host..."
+      },
+      {
+        "command": "zowe-native-proto-vsce.uninstall",
+        "title": "Zowe-SSH: Uninstall Zowe Server from Host..."
```

Comment:
If we can't find the server, should we assume that it was uninstalled?

I guess I'm curious if we can suppress this warning pop-up 😋 

<img width="219" alt="image" src="https://github.com/user-attachments/assets/3ddb7839-7b67-49a2-9e1b-4f84cc6084b4" />

Reply:
Resolved in https://github.com/zowe/zowe-native-proto/pull/34/commits/298f1f92f1df600fae04fadc60fb97fe2894914a